league/dashboard/models.py

```diff
@@ -3,35 +3,97 @@
 
 from enum import Enum
 
-from league.database import Column, Model, SurrogatePK, db, reference_col
+from league.database import (Column, Model, SurrogatePK, association_proxy, db,
+                             reference_col, relationship)
 
 Color = Enum('Color', 'white black')
 
 
+class Player(SurrogatePK, Model):
```

Comment:
RE inconsistencies, that should theoretically be a non-issue if you never touch the underlying properties and only work with the proxies (I think). You would in fact just create a `Game(SomePlayer, SomeOtherPlayer).save()` and I believe everything will be fine. 

Actually, even in the scenario you describe, I think things would be okay, but I'm having trouble fully conceptualizing it. I'll try to write a model test later to address this